polaris-core/src/main/java/org/apache/polaris/core/storage/aws/AwsStorageConfigurationInfo.java

```diff
@@ -149,6 +156,12 @@ public URI getEndpointUri() {
     return endpoint == null ? null : URI.create(endpoint);
   }
 
+  /** Returns a flag indicating whether path-style bucket access should be forced in S3 clients. */
+  @JsonIgnore
+  public boolean forcePathStyleAccess() {
```

Comment:
Nit:
```suggestion
  public boolean getPathStyleAccess() {
```
... all other functions are `get`ters